packages/core/src/actions/account/public/get-sub-account-sequences.ts

```diff
@@ -0,0 +1,22 @@
+import { CosmWasmClient } from '@cosmjs/cosmwasm-stargate'
+import {
+  ManagerQueryClient,
+  VersionControlTypes,
+} from '../../../codegen/abstract'
+import { getManagerQueryClientFromApi } from './get-manager-query-client-from-api'
+
+export async function getSubAccountSequences(
+  accountId: VersionControlTypes.AccountId,
+  cosmWasmClient: CosmWasmClient,
+  apiUrl: string,
+  ...params: Parameters<typeof ManagerQueryClient.prototype.subAccountIds>
+) {
+  const managerQueryClient = await getManagerQueryClientFromApi(
+    accountId,
+    cosmWasmClient,
+    apiUrl,
+  )
+  const { sub_accounts } = await managerQueryClient.subAccountIds(...params)
+
+  return sub_accounts
```

Comment:
Would be good to transform these into Abstract Account Ids

Reply:
There is `get-sub-account-ids` that does that.

The original class however had a separate method for resolving sequences.